spec/requests/submissions_spec.rb

```diff
@@ -377,6 +383,152 @@
+          end
+
+          context 'when evaluator recuses' do
+            it 'becomes not_started' do
+              assignment.update!(status: :recused)
+              expect(submission.evaluation_status).to eq('not_started')
+            end
+          end
+        end
+
+        context 'when all evaluations are completed' do
+          let!(:assignment) do
+            create(:evaluator_submission_assignment,
+              submission: submission,
+              evaluator: evaluator,
+              status: :assigned)
+          end
+
+          before do
+            create(:evaluation,
+              evaluator_submission_assignment: assignment,
+              submission: submission,
+              completed_at: Time.current)
+          end
+
+          it 'is completed' do
+            expect(submission.evaluation_status).to eq('completed')
+          end
+
+          context 'when new evaluator is assigned' do
+            it 'becomes in_progress' do
+              create(:evaluator_submission_assignment,
+                submission: submission,
+                evaluator: second_evaluator,
+                status: :assigned)
+              expect(submission.evaluation_status).to eq('in_progress')
+            end
+          end
+
+          context 'when only evaluator completes evaluation and then recuses' do
+            it 'transitions from completed to not_started' do
+              expect(submission.evaluation_status).to eq('completed')
+              assignment.update!(status: :recused)
+              expect(submission.evaluation_status).to eq('not_started')
+            end
+          end
+        end
+
+        context 'when multiple evaluators are assigned' do
+          let!(:first_assignment) do
+            create(:evaluator_submission_assignment,
+              submission: submission,
+              evaluator: evaluator,
+              status: :assigned)
+          end
+
+          let!(:second_assignment) do
+            create(:evaluator_submission_assignment,
+              submission: submission,
+              evaluator: second_evaluator,
+              status: :assigned)
+          end
+
+          it 'is in_progress when one evaluation is completed' do
+            create(:evaluation,
+              evaluator_submission_assignment: first_assignment,
+              submission: submission,
+              completed_at: Time.current)
+            expect(submission.evaluation_status).to eq('in_progress')
+          end
+
+          it 'is completed when all evaluations are completed' do
+            create(:evaluation,
+              evaluator_submission_assignment: first_assignment,
+              submission: submission,
+              completed_at: Time.current)
+            create(:evaluation,
+              evaluator_submission_assignment: second_assignment,
+              submission: submission,
+              completed_at: Time.current)
+            expect(submission.evaluation_status).to eq('completed')
```

Comment:
all these specs are good coverage of `submission.evaluation_status`, but they are testing the model only. none of them trigger requests so they should be moved to `spec/models/submission_spec.rb` instead.